babel.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env"

```

Comment:
Could you explain why this is included?

Reply:
By mistake I think. I've been following tutorials and courses and I believe I installed it by a misunderstanding and forgot to delete it.